### ISSUE TITLE
[Bug] Sync is stuck at 97.3%

### ISSUE THREAD

**Issue by AlexC** [getinsomnia/insomnia#1458]
- Insomnia Version: 6.4.1
- Operating System: Ubuntu 16.04

## Details

I have multiple workspaces and our team has just signed up to Insomina Teams, however the sync for our workspace is stuck at 97.3%. I've tried logging out and logging in again however the same issue.

Unsure what other details I can get to help debug this.

**Comment by gschier** [comment]
I have checked out your team's account on my end and everything looks okay from here. It's difficult to say what the issue is without having access to your machine. Can you try duplicating the workspace (from the workspace settings) to see if a fresh instance of the workspace has the same problem?

I should also mention there is a new beta sync system included in v6.4.0 that is available as an opt-in beta. The new system operates more like Git in order to provide a better experience for teams (also with less magic). https://support.insomnia.rest/article/67-version-control

**Comment by AlexC** [comment]
Thanks for the quick reply @gschier. I tried the clone and that failed too (at 95%). Out of interest I create a new workspace with a single call in and that failed at 75%.

The new beta sync however seems to work quite nicely! Looking forward to that being on by default.

**Comment by gschier** [comment]
That's quite strange. Is there anything out of the ordinary about that workspace? For example, are there many requests or large request bodies?

**Comment by AlexC** [comment]
They are all rather normal. The main one that I was trying to sync had about 30 API calls in it, all with no or little request bodies. There was a few folders to organise things, but apart from that it was normal.

As mentioned I created a new workspace with just a single call in it (a simple GET) which failed at 75%.

**Comment by mdeboer** [comment]
Same here with v6.5.3 on Mac OSX (installed using homebrew) and it's stuck at 97.9%. This only happens when disabling "Sync Cookie Jars" and/or "Sync SSL Client Certificates". When I enabled them both and synced, it jumped straight to a 100%.

My guess it was the cookie jars option for me as the developer console said: 
```
[sync] Skipping pushing blacklisted cookie jar jar_6604de37712e99058e6c3e3445f9259e6671486c
[sync] Pulling with 48 resources
```
But it got stuck there, never getting to the point where it said: `[sync] Push created 1 resources`. It did however when I synced with the options enabled.
